### DISCUSSION TITLE
Detailed site descriptions examples

### DISCUSSION THREAD

**Discussion by gpb3037** [earthlab-education/Earth-Analytics-AY24#346] [General]
Mentioned in the discussion on Wednesday afternoon 11 Sep were some concepts for detailed site descriptions to accompany our first (and subsequent) map postings.  I don't see the zoom recording at https://canvas.colorado.edu/courses/109233/external_tools/16942 as of yet, but maybe someone can point me to a good sample.

**Comment by gpb3037** [comment]
I figured out a ucb-map.md for the info, but not sure if you can call a .md as a subroutine somehow so I just cut and pasted it into the index.md - also figured out that the code is grabbing all OSM university entities in Boulder when selecting by address, vs just osmid way	4622610 for UCB.

**Comment by nquarder** [comment]
Hey @gpb3037 and great question. We'll get the recordings from this week's discussions posted soon to Canvas so stay tuned for that. 

I can point you to a couple great examples of site descriptions courtesy of @ZacharyCJDecker and @annaposlednik. If you look in the posts/01-first-map directory in the main branch of the course repo (https://github.com/earthlab-education/Earth-Analytics-AY24/tree/main/posts/01-first-map) you will see examples of their site descriptions in the subdirectories with their names. 

Anna's work: https://github.com/earthlab-education/Earth-Analytics-AY24/blob/main/posts/01-first-map/Anna_Poslednik/cub-map.md
Zack's work: https://github.com/earthlab-education/Earth-Analytics-AY24/blob/main/posts/01-first-map/ZacharyCJDecker/NCF-map.md

There are some others that haven't been merged into the main branch yet that are also great (cc: @BioNCphilic)
Randi's work: https://github.com/earthlab-education/Earth-Analytics-AY24/blob/BioNCphilic-contrib/posts/01-first-map/BioNCphilic/scc-map.md

I picked these because they include a header, short description, and a linked citation. Great job y'all! -NQ

**Reply by gpb3037** [reply]
Thanks - I think I now have figured out how to make files in the same manner under a subdirectory posts then 01-first-map, but I still don't grok how to add that to my index.md file.  I guess I missed that as the way to do the submission.

I can call the subdirectory to get the html, but I can't seem to call the .md as a sub reference.  The only way I can get the .md stuff to work is to put it in the index.md file

**Comment by eculler** [comment]
Thanks for your patience, @gpb3037 .  I'm not sure what you mean by "calling the .md as a sub reference" -- can you clarify what you're trying to do? Do you mean linking to another file? If that's the case, you can put the relative path to your file in (with the .md extension) and GitHub Pages should automatically render that link to the online version. For example, in `index.md`, you can do something like:

```markdown
[View my Site Description](./my-site-description.md) 
```

You are also welcome to keep your portfolio posts on the same page like you suggest here. You can link to a section in a page by sticking `#section-name` at the end of your URL.

I am still working on the best way for folks to submit portfolio posts so you can get and give peer feedback. My apologies that it's a bit confusing right now, but I hope we can iron out the kinks in the next week or two!
